python/GafferUI/ReferenceUI.py

```diff
@@ -157,14 +157,14 @@ def _waitForFileName( initialFileName="", parentWindow=None ) :
 	path = dialogue.waitForPath( parentWindow = parentWindow )
 
 	if not path :
-		return ""
+		return None
 
-	return str( path )
+	return path.path()
```

Comment:
```AttributeError: 'FileSystemPath' object has no attribute 'path'```

I think this needs to be `standardPath()` after the tweak we made on the other PR.

Reply:
Ack! Blindly copying my old commit faux pas. Fixed in https://github.com/GafferHQ/gaffer/pull/5071/commits/1f138bf1c6f7814e7161244f39792cc622dc08e9